kernel/console/kprintf.c

```diff
@@ -45,11 +44,11 @@ uint64_t kprintf(const char *s, ...){
 		
 		idx_in++;
 		
-		if (idx_out == 63){
-			out_str[63] = '\0';
+		if (idx_out == KPRINTF_LEN){
+			out_str[KPRINTF_LEN] = '\0';
```

Comment:
Will this not cause buffer overrun issues?  `out_str[KPRINTF_LEN]` won't actually exist, the range of indices would be from 0 to `KPRINTF_LEN - 1` (that's why I checked for a maximum index of 63 in my original version).